src/test/java/racingGame/CarTest.java

```diff
@@ -29,12 +30,34 @@ void move_whenStrategyIsFalse_positionNotChanged() {
     @Test
     @DisplayName("여러 번 이동하면 누적 이동 거리가 올바르게 증가한다")
     void move_multipleTimes_accumulatesCorrectly() {
-        Car car = new Car();
+        Car car = new Car("car1");
 
         car.move(new AlwaysMoveStrategy());
```

Comment:
```suggestion
        car.move(()  -> true);
        car.move(()  -> false);
```
람다를 활용해 위와 같이 구현하면 굳이 AlwaysMoveStrategy, NeverMoveStrategy와 같은 클래스를 추가하지 않아도 됨
3단계에서도 피드백 남김